examples/http/web_service_performance.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * this driver monitor the performance of web services by measuring their response time.
+ * Creates a Custom Variables for each URL with its response time value:
+ *  - Label: Taken from the list of urls
+ *  - Value: The response time for each urls
+ */
+
+// This is an array of URLs to be tested for response time.
+var urls = ["www.facebook.com", "www.google.com", "www.github.com", "www.domotz.com", "www.twitter.com"];
+
+/**
+ * This function measures the response time of a given URL by making an HTTP GET request.
+ * @param {string} url  The URL to be tested.
+ * @returns {Promise} A promise that resolves with the response time in milliseconds.
+ */
+function reponseTime(url) {
+    var d = D.q.defer();
+    var start = new Date();
+    D.device.http.get(url, function (response) {
+        var end = new Date();
+        var responseTime = end - start;
+        d.resolve(responseTime);
+    });
+    return d.promise;
+}
+
+/**
+ * This function tests the response time of each URL in the urls array by calling the responseTime function for each URL
+ */
+function testUrls() {
+    var variables = [];
+    var data = [];
+    urls.forEach(function (url, i) {
+        var resultat = reponseTime(url).then(function (responseTime) {
+            var uid = "id_" + i;
```

Comment:
instead of using the url index, convert the url to base64 string and take the first 50 chars because using the index couldn't define the url